.release/ci.hcl

```diff
@@ -0,0 +1,237 @@
+schema = "1"
+
+project "consul-ecs" {
+  // the team key is not used by CRT currently
+  team = "consul-ecs"
+  slack {
+    notification_channel = "C01J8QV0EF8" # team-consul-ecs
+  }
+  github {
+    organization = "hashicorp"
+    repository = "consul-ecs"
+    release_branches = ["main"]
```

Comment:
We'll update this to also include active release branches (i.e. `release/0.5.x`) 

We should start using release branches so that it's easy to do patch releases, and I'll define that process in our release docs soon.